tasks/utils.py

```diff
@@ -152,6 +158,14 @@ def get_git_commit():
     """
     return check_output(['git', 'rev-parse', '--short', 'HEAD']).decode('utf-8').strip()
 
+def get_default_python():
+    """
+    Get the default python for the current build
+    """
+    py_runtimes = os.environ.get("PYTHON_RUNTIMES", "2")
```

Comment:
We've some places (`task/rtloader.py`, `omnibus/lib/ostools.rb`) where we're defaulting on `3`, shouldn't we do the same here?

Reply:
we probably should, yes.